zxlive/settings.py

```diff
@@ -28,14 +28,19 @@ class ColorScheme(TypedDict):
     outline: QColor
 
 
-general_defaults: Dict[str, str | QTabWidget.TabPosition | int] = {
+general_defaults: dict[str, str | QTabWidget.TabPosition | int] = {
     "path/custom-rules": "lemmas/",
     "color-scheme": "modern-red-green",
     "tab-bar-location": QTabWidget.TabPosition.North,
     "snap-granularity": '4',
     "input-circuit-format": 'openqasm',
 }
 
+font_defaults: dict[str, str | int | None] = {
+    "font/size": 13,
```

Comment:
I would suggest 11 as the default size. QT's default is 9, which is small but 13 might be a bit too large. Let me know what looks good on your computer.